.github/workflows/ci-push.yml

```diff
@@ -18,6 +18,7 @@ jobs:
     runs-on: ${{ matrix.os }}
     env:
       RACKET_EXTRA_CONFIGURE_ARGS: ""
+      PREFIX_DIR: "$HOME/work/racket"
```

Comment:
This might break if/when github changes `$GITHUB_WORKSPACE` to be something else. We should use something based on that, I think.

Reply:
I see this is the change you did throughout the file. Maybe we can use `` `realpath $GITHUB_WORKSPACE/../racketcgc` `` so that it's `sh` instead to simplify the path.